### ISSUE TITLE
[fill-ratio] sphinx documentation is missing (Trac #1198)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1192]
There is only a rather short doxygen documentation. Either the documentation should be linked to the Sphinx documentation or the index.dox file should be transformed into an index.rst file. The project's maintainer should also be added at the top of the Sphinx documentation.

Migrated from https://code.icecube.wisc.edu/ticket/1198
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "There is only a rather short doxygen documentation. Either the documentation should be linked to the Sphinx documentation or the index.dox file should be transformed into an index.rst file. The project's maintainer should also be added at the top of the Sphinx documentation.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "infrastructure", 
    "summary": "[fill-ratio] sphinx documentation is missing", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-19T18:18:33", 
    "milestone": "", 
    "owner": "mjl5147", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T17:38:57: 
* olivas changed owner from "" to "mlarson"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150820T21:40:11: kkrings commented:

> Also release notes should be linked to the documentation's front page.

**Comment by icecube-trac** [comment]
Trac update at 20150824T18:04:01: 
* olivas changed priority from "critical" to "blocker"
* olivas changed component from "cmake" to "infrastructure"

**Comment by icecube-trac** [comment]
Trac update at 20150901T13:27:17: mjl5147 changed owner from "mlarson" to "mjl5147"

**Comment by icecube-trac** [comment]
Trac update at 20160224T18:19:36: 
* mjl5147 commented:

> In changeset 142440:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142440"
> Update to actually have documentation for fill-ratio. Closes #1192.
> ```


* mjl5147 changed status from "assigned" to "closed"
* mjl5147 changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
